perception/ground_segmentation/docs/ray-ground-filter.md

```diff
@@ -26,7 +26,6 @@ This implementation inherits `pointcloud_preprocessor::Filter` class, please ref
 
 | Name                          | Type   | Description                                                                                                                                                                                                                    |
 | ----------------------------- | ------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| `base_frame`                  | string | The target frame of input points                                                                                                                                                                                               |
```

Comment:
@badai-nguyen Could you add "input_frame" and "output_frame" as params? Also, I want you to write the limitation that  "input_frame" must be fixed to base_link.